### ISSUE TITLE
Plans: Customize Nudge block when user isn't an owner of the current plan

### ISSUE THREAD

**Issue by gziolo** [Automattic/wp-calypso#8449]
When working on #8222 I found out that we have several places where we display Plan upgrade nudge blocks to the users. We should make sure that we inform users that they can upgrade their existing plan only if they purchased the current plan.

Example nudge blocks related to plan upgrade:
<img width="740" alt="screen shot 2016-10-04 at 10 50 58" src="https://cloud.githubusercontent.com/assets/699132/19075088/ad2652e2-8a43-11e6-8334-2f233ed199ac.png">
<img width="730" alt="screen shot 2016-10-04 at 10 51 05" src="https://cloud.githubusercontent.com/assets/699132/19075089/ad2894a8-8a43-11e6-97ae-5915b46d3b41.png">

Every box related to the free plan can stay as it is. This is only related to the paid plans.

I'm not sure what the best approach would be in that case. We can hide this block in that case or make sure uses goes to the plans page next (on that page we are already showing proper message).

/cc @rralian @artpi

**Comment by artpi** [comment]
I think this problem will resolve itself since those huge blocks are going away :)
https://github.com/Automattic/wp-calypso/pull/8403
TLDR: they are costing us $$$$
On the old nudges there is no place for this kind of information. They link to /plans and this information can be derived fom there.

**Comment by gziolo** [comment]
Feel free to close this one when those blocks are removed :D

**Comment by gziolo** [comment]
Closing it since there was no activity for the last year.
